src/borg/platform/windows.pyx

```diff
@@ -16,22 +16,22 @@ cdef extern from 'windows.h':
 
 @lru_cache(maxsize=None)
 def uid2user(uid, default=None):
-    return default
+    return "root"
 
 
 @lru_cache(maxsize=None)
 def user2uid(user, default=None):
-    return default
+    return 0
 
 
 @lru_cache(maxsize=None)
 def gid2group(gid, default=None):
-    return default
+    return "root"
 
 
 @lru_cache(maxsize=None)
 def group2gid(group, default=None):
-    return default
+    return 0
```

Comment:
guess there should be an open ticket about this temporary hack, which could also be linked from here.

Reply:
https://github.com/borgbackup/borg/issues/7192